go/database/mpt/io/live.go

```diff
@@ -75,8 +66,9 @@ type mptStateVisitor interface {
 }
 
 type exportableArchiveTrie struct {
-	trie  *mpt.ArchiveTrie
-	block uint64
+	directory string
+	trie      *mpt.ArchiveTrie
```

Comment:
If the `ArchiveTrie` now contains the `directory` and it can be obtained from it using its `Directory()` member function, storing the `directory` explicitly here is not really necessary, right?